lib/paperclip/attachment.rb

```diff
@@ -509,25 +509,34 @@ def post_process(*style_args) #:nodoc:
     end
 
     def post_process_styles(*style_args) #:nodoc:
-      post_process_style(:original, styles[:original]) if styles.include?(:original) && process_style?(:original, style_args)
-      styles.reject{ |name, style| name == :original }.each do |name, style|
-        post_process_style(name, style) if process_style?(name, style_args)
+      styles_to_process = styles.select { |name, style| process_style?(name, style_args) }
```

Comment:
Unused block argument - `style`. If it's necessary, use `_` or `_style` as an argument name to indicate that it won't be used.<br>Line is too long. [90/80]